examples/action_dann_lightn/config.py

```diff
@@ -16,15 +16,15 @@
 # Dataset
 # -----------------------------------------------------------------------------
 _C.DATASET = CN()
-_C.DATASET.ROOT = "I:/Datasets/EgoAction/"  # '/shared/tale2/Shared'
-_C.DATASET.SOURCE = "EPIC"  # dataset choices=['EPIC', 'GTEA', 'ADL', 'KITCHEN']
-_C.DATASET.SRC_TRAINLIST = "epic_D1_train.pkl"
-_C.DATASET.SRC_TESTLIST = "epic_D1_test.pkl"
-_C.DATASET.TARGET = "EPIC"  # dataset choices=['EPIC', 'GTEA', 'ADL', 'KITCHEN']
-_C.DATASET.TAR_TRAINLIST = "epic_D2_train.pkl"
-_C.DATASET.TAR_TESTLIST = "epic_D2_test.pkl"
-_C.DATASET.IMAGE_MODALITY = "rgb"  # mode choices=['rgb', 'flow', 'joint']
-_C.DATASET.NUM_CLASSES = 8
+_C.DATASET.ROOT = 'I:/Datasets/EgoAction/'  # '/shared/tale2/Shared'
+_C.DATASET.SOURCE = 'EPIC'  # dataset choices=['EPIC', 'GTEA', 'ADL', 'KITCHEN']
```

Comment:
Why change to sigle quotes here?

Reply:
That's true and not just here.
Also what happens to your black linting, which should change all single quotes to double quotes as we discussed before
https://black.readthedocs.io/en/stable/the_black_code_style.html#strings
Did you change the config?

Reply:
I didn't want to change the black config. I only ran `pre-commit run --all` before commit, but I am not sure if I changed by mistake.
I checked `digits_dann_lightn` in the `master` branch. The `config.py` and `main.py` still use single quotes. However, they are double quotes in the `action_dann_lightn` branch. Is something wrong?

Reply:
Thanks for bringing that to my attention.
-  You should install the pre-commit hooks via `pre-commit install` at the root rather than manually run it before commit. Check out the [contributing guideline](https://github.com/pykale/pykale/blob/master/.github/CONTRIBUTING.md#before-pull-requests-pre-commit-hooks)
- It seems that black is not able to detect python files under that folder, which is weird and I have not figured out why:
 ```sh
   black .\examples\digits_dann_lightn\
   No Python files are present to be formatted. Nothing to do
  ```
However,
  ```sh
   black .\examples\digits_dann_lightn\
   No Python files are present to be formatted. Nothing to do
  ```  ```

Reply:
I think I figured out the bug: digits have `git` and black excludes `.git` by default so this folder is excluded. If I rename it, it will be blacked!

Reply:
I ran `pre-commit install`, `pre-commit run --all` and then committed via GitHub desktop or PyCharm. 
I change a double quote to a single quote and test on both `ActionDA-Pipeline` and `master` branches. When I run `pre-commit run`, black shows pass, but nothing changes. When I run `black .`, black will change this single quotes. Have you met this problem before?

Reply:
pre-commit hooks should run auto. It could be hard to resolve/debug this via messaging here so we can discuss this issue when we meet or you set up one with Shuo to find out ways you are doing differently.

Reply:
No more problems are found about black. Thanks!